doc/sphinx/addendum/type-classes.rst

```diff
@@ -452,6 +452,12 @@ Summary of the commands
    + Use the :cmd:`Typeclasses eauto` command to customize the behavior of
      this tactic.
 
+   :n:`{| bfs | dfs }`
+     Specifies whether to use depth-first search or breadth-first search.
```

Comment:
```suggestion
     Specifies whether to use breadth-first search or depth-first search.
```

Reply:
Ah, I swapped the order of the correct one before whoops.